codesyncapp/codesync-vs
# System level syncignore to ignore directories like node_modules etc

We are now using a generic `syncignore` on system level that will ignore pre-defined patterns like `node_modules`, ".dist/` etc. It has generic patterns for almost top 10 programming languagues and Linux/MacOS/Windows as well. 

**Testing instructions**
1. Make sure `syncignore.yml` is created in system directory after the plugin is activated. Verify there are no errors in downloading the file from s3
2. Addtionally `.syncignore` on repo level also matters, however if user tries something like `!node_modules` in repo's 
`.syncignore` to include this repo, it won't be included
3- Verify `globSync` is ignoring all default patterns and only condering syncable paths
4- Verify that changes to any syncignored file are ignored
